src/screens/ArticlesList/index.js

```diff
@@ -1,11 +1,34 @@
-import React from 'react';
-import {View, Text} from 'react-native';
+import React, { useState, useEffect } from 'react'
+import { View, FlatList } from 'react-native'
+import ArticleCard from 'components/content/ArticleCard'
+import appApiClient from '../../api/appApiClient';
 
+export default function Content({ navigation}) {
+const [blog, setBlog] = useState([]);
```

Comment:
why blog name? if u get articles

Reply:
I had used article and articles so many times, just decided to go for sth for less confusion in my files